src/sass/_utils.scss

```diff
@@ -84,7 +84,7 @@
 
 .sg-visually-hidden {
   position: absolute;
-  width: 1px;
+  width: 0px;
```

Comment:
could you please explain?

Reply:
It was supposed to remove this tiny little "pimple" from Link focus area:
<img width="208" alt="Screenshot 2022-11-03 at 12 04 27" src="https://user-images.githubusercontent.com/60467496/199705322-f57f2d2a-c4ea-4553-9056-01fce2113573.png">
But I figured out how to fix it in a different way.